src/Microsoft.Azure.WebJobs.Host/Executors/DefaultStorageCredentialsValidator.cs

```diff
@@ -16,8 +15,9 @@ namespace Microsoft.Azure.WebJobs.Host.Executors
     internal class DefaultStorageCredentialsValidator : IStorageCredentialsValidator
     {
         private readonly HashSet<StorageCredentials> _validatedCredentials = new HashSet<StorageCredentials>();
+        private StorageCredentials _primaryCredentials = null;
 
-        public async Task ValidateCredentialsAsync(IStorageAccount account, CancellationToken cancellationToken)
+        public async Task ValidateCredentialsAsync(IStorageAccount account, CancellationToken cancellationToken, bool isPrimaryAccount)
```

Comment:
yep, I was setting up isPrimaryAccount as a parameter with default = false, but as it turns out I still needed to update every call so it isn't worth it.  I'll make this switch